007-OFEP-transaction-context-propagation.md

```diff
@@ -1,18 +1,20 @@
-## NAME
+## Transaction Context Propagation
 
 ## State: DRAFTING
 
-Evaluation context may be used to conditionally control the value returned from a flag evaluation. You could, for example, enable a feature for all users with a specific email domain. Using evaluation context propagation, application developers can set evaluation context where it's convenient (e.g. an auth service) and have it persist for the length of a transaction (e.g. a request or thread).
+Evaluation context is a container for all contextual data that can be used used to conditionally control the value returned from a flag evaluation. Traditionally, a developer would be responsible for explicitly defining evaluation context during flag evaluation.
+
+Transaction context, a subset of the evaluation context, is a container for transaction specific data (e.g. user id, user agent, IP) that lives for the duration of a transaction. With transaction context propagation, a developer can set transaction context where it's convenient (e.g. an auth service) and have it persist for the length of a transaction (e.g. a request or thread). OpenFeature will then merge transaction context with evaluation context before flag evaluation.
 
 ## Background
 
-Many languages provide a mechanism for storing data for the length of a single transaction. This can be used to store evaluation context that can be transparently merged before flag evaluation. The implementation will vary by technology. For example, Java may use [thread local][thread-local], Golang may use [Go Context][go-context], and Node may use [async hooks][async-hooks].
+Many languages provide a mechanism for storing data for the length of a single transaction. This can be used to store transaction context that can be automatically merged with the evaluation context before flag evaluation. The implementation will vary by technology. For example, Java may use [thread local][thread-local], Golang may use [Go Context][go-context], and Node may use [async hooks][async-hooks].
```

Comment:
```suggestion
Many languages provide a mechanism for storing data for the length of a single transaction. This can be used to store transaction context that can be automatically merged with the evaluation context before flag evaluation. The implementation will vary by technology. For example, Java may use [ThreadLocal][thread-local], Golang may use [Go Context][go-context], and Node may use [async hooks][async-hooks].
```
Nitpick.